colabs/wandb-model-registry/New_Model_Logging_in_W&B.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    import wandb
</pre><p>same thing with imports. nit: would be nice to consolidate all imports to the top </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>